src/framework/config/core.clj

```diff
@@ -1,18 +1,30 @@
 (ns framework.config.core
   (:require
-    [com.stuartsierra.component :as component]
-    [config.core :refer [load-env]]))
+   [clojure.edn :as edn]
+   [clojure.java.io :as io]))
 
-(defrecord Config
-  []
-  component/Lifecycle
-  (start [_this]
-         (load-env)))
+;; default edn file
+(def default-edn-file
+  (.getAbsolutePath (java.io.File. "config/dev/config.edn")))
```

Comment:
It would be a good idea to have the default .edn file configured as an environment variable.

Reply:
Good point, I agree.

Working on it.

Reply:
Please see: https://github.com/Flexiana/framework/pull/87/commits/b349bfc7dd46ec0532e0bc089c8ab26cee253e22